### PR TITLE
Added  parameter updatedAt to UserAddress

### PR DESCRIPTION
**Pull request by vaidadry**

### CODE REVIEW

**Comment by vytautasgimbutas** [comment]
sup?

**Comment by gilbertparreno** [comment]
Looks good to me.

File: src/main/kotlin/com/paysera/lib/mokejimai/entities/UserAddress.kt [hunk 1]
```diff
@@ -11,5 +11,6 @@ data class UserAddress(
     var legacyCityName: String?,
     var streetName: String,
     var houseNumber: String,
-    var apartmentNumber: String
+    var apartmentNumber: String,
+    var updatedAt: Int?
```
Comment (Aurimas-Zarskis) [inline src/main/kotlin/com/paysera/lib/mokejimai/entities/UserAddress.kt:15]: I think this should be Date? type

Comment (gilbertparreno) [reply src/main/kotlin/com/paysera/lib/mokejimai/entities/UserAddress.kt:15]: oh yes, Vaida you should use DateSerializer, I think we already have that class.

Comment (vaidadry) [reply src/main/kotlin/com/paysera/lib/mokejimai/entities/UserAddress.kt]: just found out about this one, changed it! Thanks! ;)

**Pull request merged**